### DISCUSSION TITLE
Arrastar o debug no vscode

### DISCUSSION THREAD

**Discussion by arkanael** [balta-io/2801#61] [Perguntas e Respostas] [answered]
Vendo a aula sobre debug no .net utilizando o vscode fiquei com uma dúvida, como faço para arrastar o debug para cima (voltar o debug igual no vscommunity)

**Comment by andrebaltieri** [comment, marked as answer]
Basta clicar na seta amarela! Mas dependendo de onde o cursor está já não dá para voltar mais... O VS é mais poderoso para debug que o Code, pelo menos no C#!
